packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -109,29 +110,35 @@ The type of OAuth 2.0 flow depends on what kind of client that you're building.
 
-#### Does your client use the redirect or embedded authentication model?
+#### Does your client use the redirect or embedded model?
 
-If your client application is a SPA or a native application that redirects its authentication requests to an Okta-hosted sign-in page ([the redirect model](/docs/concepts/redirect-vs-embedded/#redirect-authentication)), use the [Authorization Code flow with PKCE](#authorization-code-flow-with-pkce). If your application hosts the authentication flow itself ([the embedded model](/docs/concepts/redirect-vs-embedded/#embedded-authentication)), use the [Interaction Code flow](#interaction-code-flow).
+[The redirect model](/docs/concepts/redirect-vs-embedded/#redirect-authentication)
+
+Use the [Authorization Code with PKCE](#authorization-code-flow-with-pkce) flow if your SPA or native app redirects authentication requests to an Okta-hosted sign-in page.
+
+[The embedded model](/docs/concepts/redirect-vs-embedded/#embedded-authentication)
+
+Use the [Interaction Code flow](#interaction-code-flow) if your app hosts the authentication flow itself.
 
 #### Does the client have an end user?
 
-If your client application runs on a server with no direct end user, then it can be trusted with its own credentials and to use them responsibly. If your client application is only doing machine-to-machine interaction, then you should use the [Client Credentials flow](#client-credentials-flow).
+A client app that runs on a server with no direct end user can be trusted with its own credentials. The app can be trusted to use them responsibly. If your client app is only doing machine-to-machine interaction, then you should use the [Client Credentials flow](#client-credentials-flow).
 
 #### Is your app high-trust?
 
-An application is **high-trust** if you own both it and the resource that it accesses. Because you own both, you can trust the application to handle your end users' usernames and passwords. In this case, and _only if other flows aren't viable_, you can use the [Resource Owner Password flow](#resource-owner-password-flow). However, it isn't possible to use this flow with multifactor authentication, so consider alternatives such as the [Authorization Code](/docs/guides/implement-grant-type/authcode/main/) or [Interaction Code flow](#interaction-code-flow).
+An app is **high-trust** if you own both it and the resource that it accesses. Because you own both, you can trust the app to handle your end users' usernames and passwords. In this case, and _only if other flows aren't viable_, you can use the [Resource Owner Password flow](#resource-owner-password-flow). However, it isn't possible to use this flow with multifactor authentication, so consider alternatives such as the [Authorization Code](/docs/guides/implement-grant-type/authcode/main/) or [Interaction Code flow](#interaction-code-flow).
```

Comment:
```suggestion
An app is **high-trust** if you own it and the resource that it accesses. Because you own both, you can trust the app to handle your end users' usernames and passwords. In this case, and _only if other flows aren't viable_, you can use the [Resource Owner Password flow](#resource-owner-password-flow). However, it isn't possible to use this flow with multifactor authentication, so consider alternatives such as the [Authorization Code](/docs/guides/implement-grant-type/authcode/main/) or [Interaction Code flow](#interaction-code-flow).
```

Reply:
"Both" feels weird before "it"...I think it's fine to leave out in this sentence because you use it in the next one